src/bika/lims/content/worksheettemplate.py

```diff
@@ -100,16 +99,6 @@
         allowed_types=("Method",),
         relationship="WorksheetTemplateMethod",
         referenceClass=HoldingReference,
-        widget=SelectionWidget(
-            format="select",
-            label=_("Method"),
-            description=_(
-                "Restrict the available analysis services and instruments"
-                "to those with the selected method."
-                " In order to apply this change to the services list, you "
-                "should save the change first."
-            ),
-        ),
     ),
```

Comment:
Removing this widget makes the filtering of instruments when a method is selected to not work at all. The reason is that the javascript in charge of updating the list of methods expects the control to be a `SelectionWidget` here: https://github.com/senaite/senaite.core/blob/2.x/src/senaite/core/browser/static/js/bika.lims.worksheettemplate.js#L11